node/Metrics.hpp

```diff
@@ -139,6 +139,38 @@ extern prometheus::simpleapi::counter_metric_t db_get_network_list;
 extern prometheus::simpleapi::counter_metric_t db_member_change;
 extern prometheus::simpleapi::counter_metric_t db_network_change;
 
+        // Fragmentation Metrics
```

Comment:
very minor comment but the indenting is different here than above. 
maybe we need to add clang format to ci

Reply:
`make clang-format` exists as of a recent commits